Upendo.Modules.DnnPageManager/ClientApp/src/app/components/manage-pages/manage-pages.component.html

```diff
@@ -10,142 +10,238 @@
+        <td mat-cell *matCellDef="let element">
+          <mat-icon *ngIf="element.IsIndexed" align="center" title="Page index disabled" class="c-default">zoom_in
+          </mat-icon>
+          <mat-icon *ngIf="!element.IsIndexed" align="center" title="Page index enabled" class="c-default">zoom_out
+          </mat-icon>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="visibility">
+        <th mat-header-cell *matHeaderCellDef></th>
+        <td mat-cell *matCellDef="let element">
+          <mat-icon *ngIf="element.IsVisible" (dblclick)="updateVisibility(element, false)" title="Hide in Menu">
+            visibility</mat-icon>
+          <mat-icon *ngIf="!element.IsVisible" (dblclick)="updateVisibility(element, true)" title="Show in Menu">
+            visibility_off</mat-icon>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="accessibility">
+        <th mat-header-cell *matHeaderCellDef></th>
+        <td mat-cell *matCellDef="let element">
+          <mat-icon *ngIf="element.HasBeenPublished" title="Page has been published" class="c-default">check_circle
+          </mat-icon>
+          <mat-icon *ngIf="!element.HasBeenPublished" title="Page not been published" class="c-default">
+            radio_button_unchecked</mat-icon>
+        </td>
+      </ng-container>
+
+
+      <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
+      <mat-row *matRowDef="let element; columns: displayedColumns;">
+      </mat-row>
+    </table>
+
+    <div ngClass="paginator">
+      <div ngClass="mat-paginator">
+        <div style="display: flex; align-items: baseline">
+          <div style="margin-right: 8px">Go to page:</div>
+          <mat-form-field appearance="legacy" style="width: 5vw">
+            <input type="number" matInput [(ngModel)]="goToPage" (keydown.enter)="updateGoToPage()"
+              autocomplete="off" />
+          </mat-form-field>
+        </div>
       </div>
+      <div>
+        <mat-paginator [length]="total$ | async" [pageSizeOptions]="[10, 25,50,100]" showFirstLastButtons
+          aria-label="Select page of periodic elements">
+        </mat-paginator>
+      </div>
+      <!-- <mat-paginator [length]="total$ | async" [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator> -->
     </div>
-    <div>
-      <mat-paginator [length]="total$ | async"
-        [pageSizeOptions]="[10, 25,50,100]"
-        showFirstLastButtons
-        aria-label="Select page of periodic elements"
-      >
-      </mat-paginator>
-    </div>
-  <!-- <mat-paginator [length]="total$ | async" [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator> -->
-</div>
 
-<span class="error-message" *ngIf="errorMessage.length > 0">
-  {{ errorMessage }}
-</span>
\ No newline at end of file
+    <span class="error-message" *ngIf="errorMessage.length > 0">
+      {{ errorMessage }}
+    </span>
+  </div>
+  <style>
```

Comment:
@alejoroman0605 Why are we putting CSS inline in the HTML file?  We should be using CSS at all times, unless there's no other way to do this.  You should be able to find a Module.css in the root of the module project.  Any necessary CSS can go there, or in another CSS file, as necessary.  If you need help on where to put the CSS, please either ask someone else on the team, or book a meeting with me and we'll review it together.

Reply:
I did it that way because when I put the media query for the small screens breakpoint in the .sass file it doesn't work. But I'm going to find a way to get it out of the html file, perhaps creating a file with a .css extension